PowerShell/ScubaGear/Rego/AADConfig.rego

```diff
@@ -666,25 +672,44 @@ tests contains {
     "RequirementMet": Status
 } if {
     DescriptionString := "global admin(s) found"
-    Conditions := [
-        count(GlobalAdmins) <= 8,
-        count(GlobalAdmins) >= 2
-    ]
-    Status := count(FilterArray(Conditions, false)) == 0
+    Status := count(FilterArray(GlobalAdminConditions, false)) == 0
 }
 #--
 
 # MS.AAD.7.2v1
 #--
 
-# At this time we are unable to test for 7.2v1
+# Save all users that don't have Global Admin role
+NotGlobalAdmins contains User.DisplayName if {
+    some User in input.privileged_users
+    not "Global Administrator" in User.roles
+}
+
+default GetScoreDescription := "No privileged users that are NOT Global Admin; Least Privilege Score = 100%"
```

Comment:
In this case, the least score is really infinite, right?  100% would be a passing score but this is a failure mode (and the condition implemented in the actual policy test will fail, as expected).

Aside: we will have to see, but there may be a case for tiny agencies where |{Global Admins}| = 1 or 2 and there is no need for any other admins.  As designed, this would fail the score.  For now, I think its fine but we may need to fix this later based on additional learnings.